modules/analytics-v2/package.json

```diff
@@ -15,7 +15,9 @@
     "@blueprintjs/core": "^3.23.1",
     "@types/node": "^10.11.3",
     "@types/react": "^16.9.19",
-    "@types/react-dom": "^16.9.5"
+    "@types/react-dom": "^16.9.5",
+    "@types/reactstrap": "^8.4.1",
+    "module-builder": "../../build/module-builder"
```

Comment:
The module builder must no longer be added as a dependency. In fact, you can remove @blueprintjs/core, "@types/node, @types/react and @types/react-dom.

They are already provided by the studio